### DISCUSSION TITLE
Javascript:function() in href add page path

### DISCUSSION THREAD

**Discussion by Dhaval-A** [requarks/wiki#6353] [Error / Bug Report] [answered]
On page named "Links" I use anchor tag where I set 
href="javascript:myFunction();"
After saving page it amend page name ->  href="/Links/javascript:myFunction();"

**Comment by Dhaval-A** [comment]
disable Sanitization under Admin > Rendering > HTML > Security. Then save your page again.
Doesn't work still adding page name as -> href="/Links/javascript:myFunction();"

**Comment by Dhaval-A** [comment]
under Admin > Rendering > HTML > Core >Treat relative links as root absolute not adding page name but still adding extra /
href="/javascript:myFunction();"

**Comment by NGPixel** [comment, marked as answer]
Use `onclick="myFunction"` instead of `href="javascript:myFunction()"`
